### ISSUE TITLE
Warning for VPN if UPnP is disabled

### ISSUE THREAD

**Issue by dsimog01** [dappnode/DNP_DAPPMANAGER#1238]
Users should be warned that VPN will not work in case that UPnP is disabled, unless they open the ports manually. This warning should be shown in the VPN tab of the dappmanager.

It is not possible to know if the ports are open from the dappmanager, so it should be enough with warning the users that if UPnP is disabled, they have to manually open the ports.

**Comment by alexpeterson91** [comment]
Agreed. The complication is that the 3(2oVPN+1WG) VPN ports aren't always an issue with UPnP. It's a sometimes issue. For me it's been always but I've done support for a long time and it's not always an issue. 

In the past I had updated the documentation and getting started guides to specifically check for the VPN ports being open in your router, and AFAIK it's been included in the new docs as well but I'll have to double check. 
Having some note in the dappmanager too would definitely help this unfortunately common issue. 
Perhaps we can come up with a safe non invasive or intrusive way to do a one time port scan from the WAN when the dyndns is first requested? Because currently dyndns is the only centralized entity linking IPs and dappnode software users. Static IP users and/or those with their own DDNS are immune from this single connection. 
Which on another note we should advertise in docs etc as and option. Lots of people have their own it's really common for people with home cloud servers and others with stuff like surveillance systems for remote view. and it's free to sign up for many/most services, ill definitely make sure I make a note of this in the docs for the most privacy conscious users.
